### ISSUE TITLE
many video url can't open

### ISSUE THREAD

**Issue by zhw-zhang** [TencentARC/VideoPainter#12]
I tried to download the data from `videovo.csv` and found that many video URLs are invalid, as they cannot even be opened in a browser. From the images, it can be seen that the downloads are not continuous. Could you please confirm this? I look forward to your help.

![Image](https://github.com/user-attachments/assets/8c7450f4-aba2-40a3-824e-07adf36b36b9)

**Comment by zhw-zhang** [comment]
anyupdate?

**Comment by mychina75** [comment]
same issue.

**Comment by yxbian23** [comment]
> I tried to download the data from `videovo.csv` and found that many video URLs are invalid, as they cannot even be opened in a browser. From the images, it can be seen that the downloads are not continuous. Could you please confirm this? I look forward to your help.
> 
> ![Image](https://github.com/user-attachments/assets/8c7450f4-aba2-40a3-824e-07adf36b36b9)

Thanks for raising this question! I will check this in one or two days~

**Comment by yxbian23** [comment]
@zhw-zhang @mychina75 @peki12345 

This issue does indeed exist. 

The problem likely stems from the videvo website's (https://www.videvo.net/) resource adjustments, which has caused some video links that were active during our data collection to now become invalid. 

Unfortunately, we currently don't have an effective solution for this issue. 

However, the good news is that the main body of our data comes from pexels (pexels.csv). After randomly checking five thousand links, we haven't found similar issues with the pexels data. 

**Therefore, we recommend that for the training data [VPData](https://huggingface.co/datasets/TencentARC/VPData), you can effectively utilize the videos with working links (all from Pexels and part from videvo). As for the Benchmark [VPBench](https://huggingface.co/datasets/TencentARC/VPBench), we have already provided all the original videos, which can be used directly.**

**Comment by zhw-zhang** [comment]
Thank you for your response. It appears that the VideoOVO dataset contains approximately 70k samples. Would it be possible to kindly provide an alternative way to download it as a packaged file (e.g., via Hugging Face)?

Additionally, may I ask if it would be possible to use the dataset directly with the training code, in the absence of the data preprocessing code (which is still pending release)?

**Comment by yxbian23** [comment]
> Thank you for your response. It appears that the VideoOVO dataset contains approximately 70k samples. Would it be possible to kindly provide an alternative way to download it as a packaged file (e.g., via Hugging Face)?
> 
> Additionally, may I ask if it would be possible to use the dataset directly with the training code, in the absence of the data preprocessing code (which is still pending release)?

Sure! I have uploaded all the videos in Videovo subset of our VPData to HF. And all the masks in VPData have been released! 

Please refer to this [section](https://github.com/TencentARC/VideoPainter#:~:text=VPBench%20and%20VPData%20Download) for detailed information!

**Comment by zhw-zhang** [comment]
That's great! I have another question: Would it be possible to directly train the model with the raw video datasets in the absence of the data preprocessing code (which is still pending release in TODO)?

**Comment by yxbian23** [comment]
> That's great! I have another question: Would it be possible to directly train the model with the raw video datasets in the absence of the data preprocessing code (which is still pending release in TODO)?

You can take a deeper look at the current dataset description~ 

The released VPData and VPBench are exactly the preprocessed data. You don't need any more data preprocessing code and you can directly train or eval your model based on VPData and VPBench.

The data preprocessing code to be released is referred to the code for constructing the VPData and VPBench.
